### PR TITLE
Update some azure deployment instructions for 1.5

### PR DESCRIPTION
**Pull request by z4ce**
Just aligns version numbers of the packages for 1.5 and service installer steps

### CODE REVIEW

File: src/deployment-guides/tko-on-azure.md [hunk 1]
```diff
@@ -250,18 +250,34 @@ sudo usermod -aG docker $USER
...
-tar -xvf tanzu-cli-bundle-linux-amd64.tar
-gzip -d kubectl-linux-v1.21.2+vmware.1.gz
 
-sudo install cli/core/v1.4.0/tanzu-core-linux_amd64 /usr/local/bin/tanzu
-tanzu plugin install --local cli all
-sudo install kubectl-linux-v1.21.2+vmware.1 /usr/local/bin/kubectl
+git clone https://github.com/z4ce/vmw-cli
+cd vmw-cli
+curl -o tmc 'https://tmc-cli.s3-us-west-2.amazonaws.com/tmc/0.4.3-fcb03104/linux/x64/tmc'
+./vmw-cli ls
```
Comment (craigtracey) [inline src/deployment-guides/tko-on-azure.md:257]: Is this extra?

Comment (z4ce) [reply src/deployment-guides/tko-on-azure.md:257]: I believe it is required the first time you use the vmw-cli to populate the cache

**Review by 2mOlaf — APPROVED** [review]
Minor issues remaining. These will not pose any real risk to the artifacts.

File: src/deployment-guides/tko-on-azure.md [hunk 2]
```diff
@@ -250,18 +250,34 @@ sudo usermod -aG docker $USER
...
+sudo install kbld-linux-amd64-* /usr/local/bin/kbld
+sudo install vendir-linux-amd64-* /usr/local/bin/vendir
+sudo install ytt-linux-amd64-* /usr/local/bin/ytt
+cd ..
+
+tanzu plugin sync
+tanzu config init
 
 # Azure CLI Install and VM Acceptance
 curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
```
[the remark this answers is not in the document]
Comment (z4ce) [reply src/deployment-guides/tko-on-azure.md:283]: this is the "manual" instructions.. non-terraform

**Review by srinivasme21 — APPROVED** [review]
IX review done and added.

**Pull request merged**